src/test/java/com/iexec/core/replicate/ReplicateServiceTests.java

```diff
@@ -68,6 +68,7 @@ class ReplicateServiceTests {
     @Mock
     private TaskLogsService taskLogsService;
 
+    @Spy
```

Comment:
Do we need this annotation ?
I thought it is needed when we manually create a class instance, which does not seem to be the case here.

Reply:
We don't need it anymore. I think I have added it to do some tests but I forgot to remove it.
FYI, it may be needed if you want to mock the tested service.